clrdefinitions.cmake

```diff
@@ -110,6 +110,10 @@ if(FEATURE_DBGIPC)
   add_definitions(-DFEATURE_DBGIPC_TRANSPORT_DI)
   add_definitions(-DFEATURE_DBGIPC_TRANSPORT_VM)
 endif(FEATURE_DBGIPC)
+if(FEATURE_GCTRIMCOMMIT_AGGRESSIVE)
+  add_definitions(-DFEATURE_GCTRIMCOMMIT_AGGRESSIVE)
```

Comment:
I do not think these things should be controlled by build defines. They should be config switches instead.